src/screens/TopicMemberScreen/index.js

```diff
@@ -14,6 +14,7 @@ import StringConstant from '../../utils/string/StringConstant';
 import UsersFragment from '../DiscoveryScreenV2/fragment/UsersFragment';
 import {Context} from '../../context';
 import NavHeader from '../TopicPageScreen/elements/NavHeader';
+import {normalize} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `normalize` from `../../utils/fonts` is added but it's not used anywhere in the provided hunks. Please ensure that it's used somewhere else in the code, otherwise, it's an unnecessary import.
```diff
- import {normalize} from '../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->